test/collection-browser.test.ts

```diff
@@ -20,6 +20,12 @@ import { analyticsCategories } from '../src/utils/analytics-events';
 import type { TileDispatcher } from '../src/tiles/tile-dispatcher';
 import type { CollectionFacets } from '../src/collection-facets';
 
+/** Wait for the next tick of the event loop */
```

Comment:
Question: This looks interesting and useful, but why is it being used and useful?
Suggest additional "why" comment.

Reply:
With collection browser being a large and complex beast, I've noticed there are some parts of it that seem to take multiple component update cycles to trickle through before a change is rendered. One example of this is the date picker, which seems to have some unusual logic that delays it from rendering in the same tick as certain updates. So in cases like that, we just need to wait an additional tick before we can reliably expect our test conditions to have rendered completely.

I don't think that's an ideal situation -- it's probably a good target for a refactor at some point. But anyway, I can add some extra comments clarifying this!